panel/src/components/Layout/Table.vue

```diff
@@ -418,6 +399,15 @@ export default {
 	height: var(--table-row-height);
 }
 
+/* Text aligment */
+.k-table-column[data-align] {
+	text-align: var(--align) !important;
```

Comment:
Would be great if we can avoid this `!important`. Is it needed if we would remove the `text-align: start;` from above?

Reply:
I already tried but unfortunately when remove the `text-align: start;`, table header centered (seems default behaviour, not via custom css)

Reply:
Like that when your suggestion applied

![captures_chrome-capture-2022-8-6 (1)](https://user-images.githubusercontent.com/3393422/188608262-8552858c-8537-4cde-8a4e-949557bb16ca.png)